LICENCE

```diff
@@ -1,4 +1,4 @@
-Copyright 2011-2012 Bulletproof Networks. All rights reserved.
+Copyright 2011-2012 Bulletproof Networks. 
```

Comment:
Are we actually Bulletproof Networks?, We should probably use "Bulletproof Group Ltd"